flocker/node/agents/cinder.py

```diff
@@ -70,6 +73,8 @@ def _run_with_logging(self, *args, **kwargs):
         with OPENSTACK_ACTION(operation=[method_name, args, kwargs]):
             try:
                 return method(*args, **kwargs)
+            except KeystoneOverLimit as e:
+                raise
```

Comment:
Removing catch and re-raising.
New log and ``retry_after`` details will come from the retry decorator i am working on currently.
Do you think it is better to log it here? Thanks!